### PR TITLE
Alembic initial setup

### PR DESCRIPTION
**Pull request by Claire56**
Created 

- model file
- database file
- alembic.ini
- elembic/env.py
- Generated an empty migration list and populated i with information 
- unable to run alembic migrate head 
```bash
sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1045, "Access denied for user 'rei_app_rds_user'@'c-98-45-159-92.hsd1.ca.comcast.net' (using password: YES)")
```

### CODE REVIEW

**Review by joshkafeero — CHANGES_REQUESTED** [review]
@Claire56 , I think you said this was a WIP, just a comment, where are the tests. 

- They'll help others know how to use these tools. We should have tests/database folder with the tests for the various components.
-  Not good to run the app to find out if the app is running. Just pytest on the test folder will give us the assurance we need that things are OK.

**Comment by Claire56** [comment]
> @Claire56 , I think you said this was a WIP, just a comment, where are the tests.
> 
> * They'll help others know how to use these tools. We should have tests/database folder with the tests for the various components.
> * Not good to run the app to find out if the app is running. Just pytest on the test folder will give us the assurance we need that things are OK.

Sure, added a connection test file, will create a testDB for detailed alembic tests , I accidentally run pytest on the actual database and it deleted the subscrition table, so expect two alembic revisions ,

File: database/testdb.py
Comment (zimba-github) [inline database/testdb.py:1]: We want out tests in a dedicated folder. We don't want test code in the code distribution. We'll leak stuff like auth details and other related issues. Also tests being able to resolve references dependencies will mean everything works well for other client components such as dagster and others.

**Review by zimba-github — APPROVED** [review]
We want out tests in a dedicated folder. We don't want test code in the code distribution. We'll leak stuff like auth details and other related issues. Also tests being able to resolve references dependencies will mean everything works well for other client components such as dagster and others. I will create a ticket for this purpose.

**Pull request merged**